src/black/handle_ipynb_magics.py

```diff
@@ -37,20 +37,16 @@
         "ESCAPED_NL",
     )
 )
-NON_PYTHON_CELL_MAGICS = frozenset(
+PYTHON_CELL_MAGICS = frozenset(
     (
-        "bash",
-        "html",
-        "javascript",
-        "js",
-        "latex",
-        "markdown",
-        "perl",
-        "ruby",
-        "script",
-        "sh",
-        "svg",
-        "writefile",
+        "capture",
+        "prun",
+        "pypy",
+        "python",
+        "python2",
```

Comment:
Python2 support has been deprecated from `black`, I think this can be removed

Reply:
Thanks, removed in https://github.com/psf/black/pull/2630/commits/744820647cc1728d0c4e82e20fcfbeaeb0fd6cad